x-pack/plugins/canvas/public/lib/flatten_panel_tree.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const flattenPanelTree = (tree, array = []) => {
```

Comment:
nit: we need to get into a habit of adding tests for utility functions as they're added.  This can be a follow-up, but I'll nag on this more and more as we move forward.

Reply:
I realized that this helper function isn't actually necessary, so I reverted these changes in ea3276d4a33d43d5d132361bd3e5d2ef57b95388.